crt_portal/cts_forms/templates/forms/complaint_view/index/filters/date.html

```diff
@@ -0,0 +1,12 @@
+{% extends "forms/complaint_view/index/filter.html" %}
```

Comment:
why is this date.html a new file here? you may have conflict with #310 which removed this and used a different layout.

Reply:
Hmm... not sure why this file exists. I haven't been changing anything with date. Maybe it came from my branching off of 621. Will delete this file.